### PR TITLE
Add local username input

### LINKED ISSUE #12: Support passing local_username
It appears there is no support for setting the `local_username` param (defined in the [deploy reporting API docs](https://explorer.docs.rollbar.com/#operation/post-deploy)).

It would be helpful to have a parameter for this (in addition to the existing `ROLLBAR_USERNAME`, which covers `rollbar_username`).

### PR DESCRIPTION
**Pull request by gavinsharp**
Fixes #12 

I chose to use an action input instead of an environment variable. It's not clear to me why `ROLLBAR_USERNAME` (or `ROLLBAR_ACCESS_TOKEN`) is expected in the environment as opposed to an action input. Would you be open to a followup input to make those action inputs instead? I imagine it would require a major version bump.

### CODE REVIEW

**Comment by gavinsharp** [comment]
> It's not clear to me why `ROLLBAR_USERNAME` (or `ROLLBAR_ACCESS_TOKEN`) is expected in the environment as opposed to an action input.

Ah, just found https://github.com/rollbar/github-deploy-action/pull/7#issuecomment-633607335. Let me see if I can clean that up...

**Comment by gavinsharp** [comment]
> Ah, just found [#7 (comment)](https://github.com/rollbar/github-deploy-action/pull/7#issuecomment-633607335).

But now I've found https://github.com/rollbar/github-deploy-action/pull/10#discussion_r435731905! 😄  Should be OK then.

Investigating what is up with the test.

**Comment by gavinsharp** [comment]
> Investigating what is up with the test.

Looks like it is just the secret not being present for the build of a fork.

File: README.md [hunk 1]
```diff
@@ -21,15 +21,16 @@ Optionally set `ROLLBAR_USERNAME` environment variable, usernames can be found a
...
-| `source_maps`  | `false`   |              | JS source map files.                             |
-| `minified_urls`| `false`   |              | Minified URLs linked to source maps above        |
+| Input           | Required  | Default      | Description                                      |
+| --------------- | --------- | ------------ | ------------------------------------------------ |
+| `environment`   | `true`    |              | The environment where the deploy is being done.  |
+| `version`       | `true`    |              | The version being deployed.                      |
+| `status`        | `false`   | `succeeded`  | The status of the deploy.                        |
+| `source_maps`   | `false`   |              | JS source map files.                             |
+| `minified_urls` | `false`   |              | Minified URLs linked to source maps above        |
+| `local_username`| `false`   |              | Username of the deploying user. Alternative to setting ROLLBAR_USERNAME |
```
Comment (mrunalk) [inline README.md:31]: Description can be "Github username of the deploying user" since this action is specific to Github.

**Review by mrunalk — CHANGES_REQUESTED** [review]
Thank you Gavin and apologies for late response. Really appreciate your contributions. Changes mostly look OK. Please find my comments. You will have to update the test.yml to reflect the new input.

File: README.md (hunk 1 above)
Comment (gavinsharp) [reply README.md:31]: Thanks for the response @mrunalk.

I think this parameter could in theory be used to specify non-GitHub usernames, right? I left it generic for that reason. But I agree that the most common use case in the context of this action would be to send GitHub usernames.

**Comment by gavinsharp** [comment]
> You will have to update the test.yml to reflect the new input.

I'm not sure how the testing really works, but will that be useful given that it already specifies `ROLLBAR_USERNAME`, which takes precedence?

File: README.md (hunk 1 above)
Comment (gavinsharp) [reply README.md:31]: Looking over the docs again, I realized there was an existing note in the `README` about usernames - I adjusted it in ebc3fcdbbbb9821f42988fa8b9de67da1147a5f9.

**Comment by gavinsharp** [comment]
> > You will have to update the test.yml to reflect the new input.
> 
> I'm not sure how the testing really works, but will that be useful given that it already specifies `ROLLBAR_USERNAME`, which takes precedence?

Made this change in 91d37f90a8c983be3fda14bf493a471430a2e80d anyhow.

**Comment by gavinsharp** [comment]
@mrunalk can you review this again? I've made the requested changes.

**Comment by mrunalk** [comment]
@gavinsharp, LGTM overall. Somehow deploy step is not able to find ROLLBAR_ACCESS_TOKEN. Probably some issue on the Github repo config. Looking into it.

**Review by mrunalk — APPROVED** [review]

**Comment by mrunalk** [comment]
Looks like forked repos don't have access to ROLLBAR_ACCESS_TOKEN. I will go ahead and merge it for now. Next time can you try creating a branch on this repo?

**Pull request merged**